InvenTree/order/api.py

```diff
@@ -1146,6 +1152,145 @@ class PurchaseOrderAttachmentDetail(AttachmentMixin, RetrieveUpdateDestroyAPI):
     serializer_class = serializers.PurchaseOrderAttachmentSerializer
 
 
+class OrderCalendarExport(ICalFeed):
+    """Calendar export for Purchase/Sales Orders
+
+    Optional parameters:
+    - include_completed: true/false
+        whether or not to show completed orders. Defaults to false
+    """
+
+    try:
+        instance_url = InvenTreeSetting.get_setting('INVENTREE_DEFAULT_CURRENCY', create=False, cache=False)
```

Comment:
This seems to be the wrong setting

Reply:
Doh..

Too much copy-paste. Hey, it worked though ;) I'll correct it.

Reply:
This is fixed.